### DISCUSSION TITLE
repeater / container custom css

### DISCUSSION THREAD

**Discussion by samg666** [Budibase/budibase#3079] [Help] [answered]
Hello,

I have put a container inside the repeater, and would like to set a hover background change, but it seems didn't work. the `cursor:pointer;` works fine, but `:hover{ background-color:yellow;}` didn't work. I am not sure if there any syntax error. Can someone help?

<img width="1647" alt="iShot2021-10-19 18 32 13" src="https://user-images.githubusercontent.com/29649651/137892888-f1928ee9-e358-456e-89b4-b4d2beefb404.png">

**Comment by mjashanks** [comment, marked as answer]
Hey Sam. This discussion might help: https://github.com/Budibase/budibase/discussions/2218

In general the custom CSS feature is a bit unreliable as it's hard to get it to apply CSS to child components. You'll need to use `.spectrum` classes and variables to achieve this.

**Reply by samg666** [reply]
Hi @mjashanks Michael, I have tried add below custom css on the container, but not work. It works if I manually add on the Chrome browser. It seems the custom css cannot change the predefined CSS class. Is there any other way to implement hover background change on the repeater?

```
.valid-container.direction-row.hAlign-left.vAlign-middle.size-shrink.gap-L.svelte-d6lr62:hover {
    background-color: red;
}
```

![iShot2021-10-21 11 54 01](https://user-images.githubusercontent.com/29649651/138209440-eb7a23f2-4746-497e-a2c5-26642abe384e.png)

**Reply by mjashanks** [reply]
Try using the css variables:

Inside my table's "Edit Custom CSS", I wrote `--spectrum-table-row-background-color-hover: red`, which worked for the row hover colour.

**Reply by khalid-al-walid** [reply]
hi, where can i find more example for custom css? i try to custom my card component. i have try google it, but found nothing.
